charts/mattermost-team-edition/README.md

```diff
@@ -7,6 +7,7 @@
 ```bash
 $ helm repo add mattermost https://helm.mattermost.com
 $ helm install mattermost/mattermost-team-edition \
+  --generate-name \
```

Comment:
@vijaynag-bs there is not `--generate-name` for helm 2

Reply:
That was supposed to helm v3. Should I change to helm v2 ??

Reply:
Sorry, I have changed. It was rectified in two lines.